workflow-bot-app/src/main/java/com/symphony/bdk/workflow/exception/ExceptionHandler.java

```diff
@@ -0,0 +1,10 @@
+package com.symphony.bdk.workflow.exception;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+
+@Component
+@ControllerAdvice
+@Slf4j
+public class ExceptionHandler extends GlobalExceptionHandler {}
```

Comment:
we may just use the GlobalExceptionHanlder.java, as it is only used by wdk webapp.